src/nautilus_librarian/typer/commands/workflows/actions/rename_base_images_action.py

```diff
@@ -0,0 +1,75 @@
+from os import makedirs, path
+from shutil import move
+
+from nautilus_librarian.domain.file_locator import (
+    file_locator,
+    guard_that_base_image_exists,
+)
+from nautilus_librarian.mods.dvc.domain.utils import extract_renamed_files_from_dvc_diff
+from nautilus_librarian.mods.namecodes.domain.filename import Filename
+from nautilus_librarian.typer.commands.workflows.actions.action_result import (
+    ActionResult,
+    ErrorMessage,
+    Message,
+    ResultCode,
+)
+
+
+class BaseImageNotFoundError(FileNotFoundError):
```

Comment:
We have to use the one we mode to the Librarian domain.

Reply:
You are right